### DISCUSSION TITLE
nohup: failed to run command '/xxx/be/lib/palo_be': No such file or directory

### DISCUSSION THREAD

**Discussion by southing-hub** [apache/doris#10681] [A - General / Q&A]
## be运行环境说明
系统：ubuntu_22.04
gcc :gcc version 10.3.0 (Ubuntu 10.3.0-15ubuntu1)
doris：1.0.0
jdk：1.8
其他的均按文档要求走

## 具体描述
本地拉取镜像apache/incubator-doris:build-env-for-1.0.0，关联本地的.m2库和源代码，
然后执行sh build.sh 进行编译，成功后上传到生产环境进行配置启动，be启动报错。错误如下：
nohup: failed to run command '/opt/software/output/be/lib/palo_be': No such file or directory

**Comment by dataalive** [comment]
你是部署到这个位置了吗？

**Reply by southing-hub** [reply]
是的。问题中的路径xxx是省略的写法，be真实的部署路径是报错信息中的/opt/software/output/be

**Comment by southing-hub** [comment]
![image](https://user-images.githubusercontent.com/32639935/178180627-3aee5018-e34b-42ee-989a-f1189300fd16.png)

**Comment by hf200012** [comment]
现在在ubuntu 22.04上启动还没适配

**Comment by dataalive** [comment]
https://github.com/apache/doris/pull/11643 
这个PR会fix这个问题。
